magento/magento2
# Fix missing escape Url method

### Description (*)

Correct missing escape url in layout template module widget

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. Go to Admin 
2. Go Content > Elements > Widgets
3. Edit one exists widget
4. Page should load without break
5. Add layout update should work

For test edit widget add/delete "Layout Updates" we need merge the PR https://github.com/magento/magento2/pull/29991 first 

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#30036: Fix missing escape Url method
